db/db.go

```diff
@@ -16,31 +16,23 @@
 package db
 
 import (
-	"context"
-	"errors"
 	"math/rand"
 	"strings"
 	"time"
 
-	"github.com/cloudwan/gohan/db/file"
 	"github.com/cloudwan/gohan/db/options"
-	"github.com/cloudwan/gohan/db/sql"
 	"github.com/cloudwan/gohan/db/transaction"
 	"github.com/cloudwan/gohan/schema"
-	"github.com/cloudwan/gohan/util"
 )
 
 //DefaultMaxOpenConn will applied for db object
 const DefaultMaxOpenConn = 100
 
-var errNoSchemasInManager = errors.New("No schemas in Manager. Did you remember to load them?")
-
 //DB is a common interface for handing db
 type DB interface {
 	Connect(string, string, int) error
 	Close()
-	Begin() (transaction.Transaction, error)
-	BeginTx(ctx context.Context, options *transaction.TxOptions) (transaction.Transaction, error)
+	Begin(options ...transaction.Option) (transaction.Transaction, error)
```

Comment:
Nitpick: I think I prefer the name `BeginTx`; `Begin` does not really tell what it begins.

Reply:
sure, done